.github/workflows/release.yml

```diff
@@ -164,6 +170,7 @@ jobs:
           go test -v -tags "quickstart automated" ./ziti/cmd/edge/...;
 
       - name: Run Unit and Integration Tests
+        if: ${{ vars.ZITI_SKIP_INTEGRATION_TESTS != 'true' }}
```

Comment:
Let forks skip the tests, e.g., to test release CI, because the tests will never succeed outside the upstream repo. They require central authorization for something related to the import/export feature.